### ISSUE TITLE
RobertaForMaskedLM support

### ISSUE THREAD

**Issue by KoichiYasuoka** [informatix-inc/bert#1]
Thank you @ifx-556 to create new Japanese RoBERTa for [MASK]-ed Language Model, but I couldn't use it with [RobertaForMaskedLM](https://huggingface.co/docs/transformers/model_doc/roberta#transformers.RobertaForMaskedLM). It seems two difficulties here:

* `IFXRoBERTa` inherits `torch.nn.Module` but not [PreTrainedModel](https://huggingface.co/docs/transformers/v4.17.0/en/main_classes/model#transformers.PreTrainedModel).
* `TextProcessor` doesn't inherit [PreTrainedTokenizer](https://huggingface.co/docs/transformers/main/en/main_classes/tokenizer#transformers.PreTrainedTokenizer) and it coerces `mojimoji.han_to_zen` for any special tokens (such as [MASK]).

I had tried some extensions for another RoBERTa model [roberta-base-japanese](https://huggingface.co/cl-tohoku/roberta-base-japanese), as described in [my diary](https://srad.jp/~yasuoka/journal/651998/), but I couldn't apply similar way for your model. Do you have any idea?
